overlord/ifacestate/udevmonitor/udevmon.go

```diff
@@ -91,8 +91,15 @@ func (m *Monitor) Connect() error {
 		return fmt.Errorf("cannot start udev monitor: %s", err)
 	}
 
-	// TODO: consider passing a device filter to reduce noise from irrelevant devices.
-	m.monitorStop = m.netlinkConn.Monitor(m.netlinkEvents, m.netlinkErrors, nil)
+	var filter netlink.Matcher
+	// TODO: extend with other criteria based on the hotplug interfaces
```

Comment:
will we need to change the list below often? is it worth consulting all interface with hotplug enable instead to make it?

Reply:
We will need to update this list whenever we introduce a new hotplug interface for a new subsystem, so probably not very often.

Yes, I had this idea very early, hotplug interfaces would declare subsystems/filters that we could plug here, but Gustavo didn't like it (perhaps I explained it in aconfusing way). I agree it would be more clean if interfaces declared their subsystems in their definitions, it's more clear and easier to whoever updates an interface to define this in a single place. I'm happy to implement something like this if we agree on the details (I think this would be a material for a followup and not critical for readiness of the feature).

Reply:
Yes, doesn't feel like a blocker for now but something to consider the next time we need to tweak the list